packages/teams-js/src/public/sharing.ts

```diff
@@ -194,4 +194,80 @@ export namespace sharing {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.sharing ? true : false;
   }
+
+  /**
+   * Namespace to get the list of content shared in a context
+   *
+   * @beta
+   */
+  export namespace history {
+    /**
+     * Represents IContentResponse parameters.
```

Comment:
>Represents IContentResponse parameters. [](http://example.com/codeflow?start=7&length=39)

Consider:

"Represents the data returned when calling {@link sharing.history.getContent}"